examples/pytorch/hetgnn/graph.py

```diff
@@ -0,0 +1,219 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+"""Graph engine implementation for HetGNN."""
+import csv
+from typing import Any, Dict, List, Tuple, Union
+
+import numpy as np
+from torch.utils.data import IterableDataset
+
+from deepgnn.graph_engine import Graph, SamplingStrategy
+
+_node_base_index = 1000000
+
+
+class MockGraph(Graph):
```

Comment:
Rename file to mockgraph.py?

Reply:
That's not java. mockgraph.MockGraph stutters.